frontend/src/app/patients/Components/ManageEmails.tsx

```diff
@@ -136,15 +144,20 @@ const ManageEmails: React.FC<Props> = ({
                 onEmailChange(idx, e.target.value)
               }
             />
-            <div className="flex-align-self-end">
-              <button
-                className="usa-button--unstyled padding-105 height-5"
-                onClick={() => onEmailRemove(idx)}
-                aria-label={`Delete email ${email}`.trim()}
-              >
-                <FontAwesomeIcon icon={"trash"} className={"text-error"} />
-              </button>
-            </div>
+            {idx > 0 ? (
+              <div className="flex-align-self-end">
+                <button
+                  className="usa-button--unstyled padding-105 height-5"
```

Comment:
I think this issue was there before but could we add a `cursor-pointer` class to this `<button>` so there are some visual indicators that the trash can is a button?